spec/Section 2 -- Language.md

```diff
@@ -47,31 +47,22 @@ match, however some lookahead restrictions include additional constraints.
 ## Source Text
 
 SourceCharacter ::
-  - "U+0009"
-  - "U+000A"
-  - "U+000D"
-  - "U+0020–U+FFFF"
+  - "Any Unicode scalar value"
```

Comment:
This is the one change I can see so far which I am not sure about.
I can see how you want to escape control character in a String, because you really want to be able to express ALL unicode scalar values. 
But why unescaped in a document itself? In 6 years GraphQL Java nobody ever wanted to that afaik.

Reply:
I think this earlier change to exclude control characters was well intentioned but ultimately more costly than helpful.

In practice I agree with your point, but there are corner cases which have caused real confusion.

First, it only excluded the ascii-range control chars, and not the control chars > 0x7F, so while the intent was to exclude control chars, in practice there were others allowed.

But I think more importantly, the real cost is being different from other languages as a source of complexity and confusion. No other modern language has this kind of source level exclusion, and this ends up being a land-mine for dynamic queries with string concatenation which need to then remember to do additional input sanitation.

Ultimately this previous attempt to exclude control chars was just a source of implementation complexity of dubious value. My intent with the change is to make Unicode source chars no different from nearly all other modern languages.

Reply:
Thanks for explaining. I am happy with it.